src/main/java/stream/flarebot/flarebot/util/ReportManager.java

```diff
@@ -60,13 +59,14 @@ public Report getReport(String guildID, int id) {
         return report[0];
     }
 
-    public List<Report> getReportsToSave(){
+    public List<Report> getReportsToSave() {
         return reportsToSave;
     }
 
     private static ReportManager instance;
-    public static ReportManager getInstance(){
-        if(instance == null){
+
+    public static ReportManager getInstance() {
+        if (instance == null) {
```

Comment:
Again, you're making changes to irrelevant code. Please uncheck the reformat and organize imports options in the commit dialog. Please.